ReClass.NET/Nodes/BaseNode.cs

```diff
@@ -39,8 +39,11 @@ public abstract class BaseNode
 		/// <summary>Gets or sets the parent node.</summary>
 		public BaseNode ParentNode { get; internal set; }
 
-		/// <summary>Gets a value indicating whether this node is wrapped into an other node.</summary>
-		public bool IsWrapped => ParentNode is BaseWrapperNode;
+		/// <summary>Gets a value indicating whether this node is wrapped into an other node. </summary>
+		public bool IsWrapped => (ParentNode is BaseWrapperNode);
```

Comment:
```suggestion
		/// <summary>Gets a value indicating whether this node is wrapped into an other node.</summary>
		public bool IsWrapped => ParentNode is BaseWrapperNode;
```
Unneeded changes?

Reply:
Yeah left overs from code I had there first. will clean it up!